### ISSUE TITLE
Uncaught Error: Parse error on line 12: Expecting 'STRING' got '}'

### ISSUE THREAD

**Issue by timelessvirtues** [AtomLinter/linter-jscs#169]
[Enter steps to reproduce below:]

1. Any time the linter runs.

**Atom Version**: 1.5.0-beta1 & 1.4.0
**System**: Mac OS X 10.11.3
**Thrown From**: [linter-jscs](https://github.com/AtomLinter/linter-jscs) package, v3.4.1

### Stack Trace

Uncaught Error: Parse error on line 12:
...ebird": "^3.1.4",  }}
----------------------^
Expecting 'STRING', got '}'

```
At /Applications/Atom Beta.app/Contents/Resources/app.asar/src/pane.js:825

Error: Parse error on line 12:
...ebird": "^3.1.4",  }}
----------------------^
Expecting 'STRING', got '}'
    at Object.parseError (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/node_modules/jsonlint/lib/jsonlint.js:55:11)
    at Object.parse (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/node_modules/jsonlint/lib/jsonlint.js:132:22)
    at exports.parse (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/node_modules/jsonlint/lib/jsonlint.js:417:53)
    at Object.exports.getContent (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:81:27)
    at Object.<anonymous> (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:142:31)
    at /Users/peter/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:36:24
    at Array.filter (native)
    at findup (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:32:25)
    at Object.exports.load (/Users/peter/.atom/packages/linter-jscs/node_modules/jscs/lib/cli-config.js:140:9)
    at Function.getConfig (/Users/peter/.atom/packages/linter-jscs/src/linter-jscs.js:155:23)
    at /Users/peter/.atom/packages/linter-jscs/src/linter-jscs.js:81:29
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1067:27)
    at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1037:19)
    at TextEditor.module.exports.TextEditor.save (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/text-editor.js:630:26)
    at Pane.module.exports.Pane.saveItem (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/pane.js:545:18)
    at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/pane.js:528:19)
    at Workspace.module.exports.Workspace.saveActivePaneItem (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/workspace.js:644:35)
    at atom-workspace.commandRegistry.add.core:save (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/register-default-commands.js:173:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:22.1.0 core:cancel (atom-notification.error.icon.icon-flame.native-key-bindings.has-detail.has-close)
     -1:19.2.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:55.1.0 core:close (atom-text-editor.editor.mini.is-focused)
  2x -0:07.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:06.9.0 core:paste (atom-text-editor.editor.is-focused)
     -0:03.0 core:cancel (atom-notification.error.icon.icon-flame.native-key-bindings.has-detail.has-close)
     -0:00.8.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "projectHome": "/Users/peter/GitHub"
  }
}
```

### Installed Packages

```coffee
# User
jshint, v1.8.3
language-json-comments, v0.16.3
linter, v1.11.3
linter-jscs, v3.4.1
minimap, v4.19.0
pretty-json, v0.4.1

# Dev
No dev packages
```

**Comment by Arcanemagus** [comment]
Does `jscs` display the same issue when ran from the terminal on the file? The error is coming from the `jscs` code.

**Comment by casesandberg** [comment]
This has been fixed, please upgrade to `linter-jscs@3.4.3`

**Comment by Arcanemagus** [comment]
Marking as a duplicate of #257.
